support/cas-server-support-oidc/src/main/java/org/apereo/cas/oidc/config/OidcConfiguration.java

```diff
@@ -463,11 +467,26 @@ public OidcIntrospectionEndpointController oidcIntrospectionEndpointController()
         return new OidcIntrospectionEndpointController(context);
     }
 
+    /**
+     * defines how the logoutURL configured at the OIDC service is matched against the given
+     * <i>post_logout_redirect_uri</i> at an OIDC RP-initiated logout.<br/>
+     * default is String::equalsIgnoreCase
+     *
+     * @return
+     *  the function that matches the given logout url against the configured logout url(s).
+     */
+    @Bean(name = POST_LOGOUT_REDIRECTURL_MATCHER_BEAN_NAME)
+    @ConditionalOnMissingBean(name = POST_LOGOUT_REDIRECTURL_MATCHER_BEAN_NAME)
+    public BiFunction<String, String, Boolean> postLogoutRedirectUrlMatcher() {
```

Comment:
This is not necessary. Why not just have the default use pattern matching?

If you want to allow flexibility into the matcher, you need to make that be a proper API and not a BiFunction.